mParticle-Apple-SDK/Include/mParticle.h

```diff
@@ -34,6 +38,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @class MPSideloadedKit;
+@class MPKitContainer;
```

Comment:
```suggestion
```
This isn't needed since you're importing the header, also the class name is different.